src/serlio/serlioPlugin.cpp

```diff
@@ -42,6 +43,7 @@ constexpr bool DBG = false;
 constexpr const char* NODE_MODIFIER = "serlio";
 constexpr const char* NODE_MATERIAL = "serlioMaterial";
 constexpr const char* NODE_ARNOLD_MATERIAL = "serlioArnoldMaterial";
+constexpr const char* CMD_CREATE_MATERIAL = "createMaterial";

```

Comment:
For consistency, the command should be called `serlioCreateMaterial`

Reply:
renamed